angular.json

```diff
@@ -60,7 +61,13 @@
                                 "type": "anyComponentStyle",
                                 "maximumWarning": "6kb"
                             }],
-                            "optimization": true,
+                            "optimization": {
+                                "scripts": false,
```

Comment:
This seems to deactivate the scripts optimization, not sure this is what we should do since it's kinda ignoring to original issue.

Reply:
Hello. Implementing the changes here, I verified that running the server and accessing the IP of the host where the Web UI runs, it is not possible to add remote servers. Do you know what can cause this? Grateful!